setup.py

```diff
@@ -8,12 +8,12 @@
     readme = f.read()
 
 setup(
-    name="tutor-s3",
+    name="tutor-contrib-s3",
```

Comment:
This rename still needs a mention in the commit message.

Reply:
I will add it